crs_motion_planning/include/crs_motion_planning/path_processing_utils.h

```diff
@@ -37,17 +37,6 @@ namespace crs_motion_planning
 bool parsePathFromFile(const std::string& yaml_filepath,
                        const std::string& waypoint_origin_frame,
                        std::vector<geometry_msgs::msg::PoseArray>& raster_strips);
-
```

Comment:
I think you can get rid of line 22 now. I think this is the only one that depends on tesseract
```
#include <tesseract_common/types.h>
```